index.html

```diff
@@ -275,7 +275,7 @@ <h1 class="form_header">  Contact me</h1>
             <div class="form_inputs">
               <input type="text" placeholder="Jhon Smith" class="form_input" maxlength="30" required name="name">
               <input type="email" placeholder="jhonedao@gmail.com" class="form_input" required name="email">
-              <textarea cols="30" rows="10" placeholder="Write your message here" maxlength="500" required class="form_input" name="message"></textarea>
+              <textarea cols="30" rows="10" placeholder="Your message goes here" maxlength="500" required class="form_input" name="message"></textarea>
```

Comment:
Max length can be more because people want to make extensive comments is necessary.

Reply:
- [ ] Kindly make your request specific and straight to the point. For instance you said 'Max length can be more because people want to make extensive comments is necessary.'. Being more specific will help avoid review loops. :wink: